plugins/cloudbuild/src/components/WorkflowRunsTable/WorkflowRunsTable.tsx

```diff
@@ -67,31 +68,33 @@ const generatedColumns: TableColumn[] = [
     title: 'Ref',
     render: (row: Partial<WorkflowRun>) => (
       <Typography variant="body2" noWrap>
-        <p>{row.substitutions?.BRANCH_NAME}</p>
+        {row.substitutions?.BRANCH_NAME}
       </Typography>
     ),
   },
   {
     title: 'Commit',
     render: (row: Partial<WorkflowRun>) => (
       <Typography variant="body2" noWrap>
-        <p>{row.substitutions?.SHORT_SHA}</p>
+        {row.substitutions?.SHORT_SHA}
       </Typography>
     ),
   },
   {
     title: 'Created',
     render: (row: Partial<WorkflowRun>) => (
-      <Typography variant="body2" noWrap>
-        <p>{moment(row.createTime).format('DD-MM-YYYY hh:mm:ss')}</p>
+      <Typography data-testid="cell-created" variant="body2" noWrap>
+        {DateTime.fromISO(row.createTime ?? DateTime.now().toISO()).toFormat(
+          'dd-MM-yyyy hh:mm:ss',
```

Comment:
I think this is a great opportunity to put some [standards](https://en.wikipedia.org/wiki/ISO_8601) in place and make this `year-month-day` as it's harder to confuse day and month :)

Reply:
I can also smell an ADR.

Reply:
I noticed a few different formats going on in some other plugins as well... this might be ok though as I'm unfamiliar with the project at the moment. 

Would it ok to follow up with an ADR, do an audit of all the plugins and change where needed as a separate task?

Reply:
Sure - happy to do that, let's create a ticket out the back of it to track it.

Reply:
Sure let's fix it in a separate PR & ADR 👍

Reply:
RFC created (#7178), I'll also draft up an ADR for it.